examples/docker.yml

```diff
@@ -60,5 +42,15 @@ files:
 trust:
   image:
```

Comment:
this is awesome!  I added an organization level key so I'll edit this to make it less wieldy